docs/tests/getting-started/github-actions.yml

```diff
@@ -24,15 +20,15 @@ jobs:
         uses: actions/checkout@v2
 
       # You need to run `dagger project init` locally before and commit the cue.mod directory to the repository with its contents
-      - name: Project update
-        uses: dagger/dagger-for-github@v2
-        with:
-          version: 0.2
-          # To pin external dependencies, you can use `project update github.com/[package-source]@v[n]`
-          args: project update
-
       - name: Deploy to Netlify
         uses: dagger/dagger-for-github@v2
         # See all options at https://github.com/dagger/dagger-for-github
         with:
-          args: do deploy
+          version: 0.2
+          # To pin external dependencies, you can use `project update github.com/[package-source]@v[n]`
+          cmds: |
+            project update
+            do deploy
+        env:
+          # Get one from https://app.netlify.com/user/applications/personal
+          NETLIFY_TOKEN: ${{ secrets.NETLIFY_TOKEN }}
```

Comment:
This secret is only necessary for the dagger action. We don't want to expose it to the whole workflow (security best practice) with a global env.